main.go

```diff
@@ -92,14 +92,15 @@ func Run(args []string) error {
 		return err
 	}
 
-	// Logging SSH mount point since SSH backend mount point at Vault server
-	// can vary and helper has no way of knowing it automatically. ssh-helper reads
-	// the mount point from the configuration file and uses the same to talk
+	// Logging namespace and SSH mount point since SSH backend mount point at Vault server
+	// can vary and helper has no way of knowing these automatically. ssh-helper reads
+	// the namespace and mount point from the configuration file and uses the same to talk
 	// to Vault. In case of errors, this can be used for debugging.
 	//
 	// If mount point is not mentioned in the config file, default mount point
 	// of the SSH backend will be used.
 	log.Printf("[INFO] using SSH mount point: %s", clientConfig.SSHMountPoint)
+	log.Printf("[INFO] using namespace: %s", clientConfig.Namespace)
```

Comment:
Don't we have to do a `client.SetNamespace` to be able to actually use the new config field?

Reply:
The client object returned from the API already has it set. I was a bit surprised to find all the config parsing and setup for this utility being doing in the API module and not here.  https://github.com/hashicorp/vault/commit/eb666e94bdb416f49ede71f4f6a4212aebb2683f#diff-5524bc23fc3211255e85bfe799553592R129

Reply:
I see. Remembered looking at it too. Yeah. This code was written a long time ago :-)